pytorch_lightning/core/lightning.py

```diff
@@ -1202,13 +1200,14 @@ def optimizer_step(self, current_epoch, batch_idx, optimizer,
 
         """
         if on_tpu:
-            xm.optimizer_step(optimizer)
+            xm.optimizer_step(optimizer, {'closure': optimizer_closure})
```

Comment:
we should check what's going on here, this is so far the only obvious place I found where we touch tpu logic
cc @lezwon

Reply:
Yeah, i'm not sure it matches the function signature of `optimizer_step`. https://github.com/pytorch/xla/blob/f572937c4da8f382b1aa8448b23563df0084ec4f/torch_xla/core/xla_model.py#L755